### ISSUE TITLE
Unit test TestNormalizePath fails on Windows

### ISSUE THREAD

**Issue by fredbi** [go-openapi/spec#63]
Failure with current master when building on Windows 10 OS (go 1.9):
all refs fragments are considered paths. with `\`  everywhere.
One would expect that only the file path gets `\` replaced.

```
expected: "/another/base/path.json#/definitions/Pet"
actual : "\another\base\path.json#\definitions\Pet" 
```
Actually expected on windows, in my opinion:
```
expected: "c:\another\base\path.json#/definitions/Pet"
```

```
C:\Users\root\go\src\github.com\go-openapi\spec>go test -v -run Normalize
=== RUN TestNormalizePaths
--- FAIL: TestNormalizePaths (0.00s)
expander_test.go:73:
Error Trace: expander_test.go:73
Error: Not equal:
expected: "/another/base/path.json#/definitions/Pet"
actual : "\another\base\path.json#\definitions\Pet"
Test: TestNormalizePaths
expander_test.go:73:
Error Trace: expander_test.go:73
Error: Not equal:
expected: "/base/another/base/path.json#/definitions/Pet"
actual : "\base\another\base\path.json#/definitions/Pet"
Test: TestNormalizePaths
FAIL
exit status 1
FAIL github.com/go-openapi/spec 0.593s
```

**Comment by mtfelian** [comment]
Fixed this in PR #67

**Comment by fredbi** [comment]
Verified. Works. Thanks for the great job
